src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
+@import "./variables";
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-day-gap;
+  width: 7 * $calendar-day-width + 16px;
+
+  &__day {
+    box-sizing: border-box;
+    height: $calendar-day-width;
```

Comment:
You can add properties such as box-sizing to normalize file or in the start of the main.scc using 

`* {
  box-sizing: border-box;
}`